### ISSUE TITLE
Missing import statements on error messages when starting without samplesheet

### ISSUE THREAD

**Issue by asp8200** [nf-core/sarek#1566]
### Description of the bug

When not supplying `--input` (directly or indirectly through use of profile) in a cmd like
```
nextflow run nf-core/sarek -r 3.4.2 -profile docker --outdir results
```
on gets the not-so-informative error msg:
```
ERROR ~ No such variable: Nextflow

 -- Check script '/home/ubuntu/.nextflow/assets/nf-core/sarek/./subworkflows/local/utils_nfcore_sarek_pipeline/main.nf' at line: 344 or see '.nextflow.log' file for more details
```

It might not be so straightforward to improve on this, as not all commands for sarek require `--input`:

https://nfcore.slack.com/archives/C05V9FRJYMV/p1718263762323759?thread_ts=1718262962.010069&cid=C05V9FRJYMV

**Comment by FriederikeHanssen** [comment]
The issue is, that `Nextflow` is improperly called in one of the utils subworkflows. Removing it and just call `error` should fix it.

**Comment by pharmlovex** [comment]
This issue has been fixed [#1712 ](https://github.com/nf-core/sarek/pull/1712).
